Lib/lib2to3/tests/data/crlf.py

```diff
@@ -1,3 +1,3 @@
-print "hi"
-
-print "Like bad Windows newlines?"
+print "hi"
+
+print "Like bad Windows newlines?"

```

Comment:
Wow, that's hilarious. We've had a \r\n file but something along the way wiped the newlines O_O